front/src/pages/AddItem/AddItem.tsx

```diff
@@ -1,28 +1,48 @@
-import React from 'react';
+import React, { useRef } from 'react';
 import styles from './AddItem.module.scss';
+import useAddTodo from '../../hooks/useAddTodo';
+import {
+  TodoImportance,
+  TodoImportanceValues,
+  TodoToCreate,
+} from '../../types';
+import { capitalize } from '../../helpers';
 
 const AddItem = () => {
+  const { mutate: addTodo } = useAddTodo(resetForm);
+  const addFormRef = useRef<HTMLFormElement>(null);
+
+  function resetForm()  {
+    addFormRef?.current?.reset();
+  }
+
   const handleFormSubmit = (e: React.SyntheticEvent) => {
     e.preventDefault();
     const target = e.target as HTMLFormElement;
-    const objectOfFields = Object.fromEntries(new FormData(target));
+    const formFields = Object.fromEntries(new FormData(target)) as {[k: string]: string} ;
```

Comment:
for learning purpose I suggest to switch onto controlled inputs in next separate MR

https://reactjs.org/docs/uncontrolled-components.html

Not really required here imo, just to avoid question "why?" from the future readers